cypress/integration/FileInput/common/index.js

```diff
@@ -8,7 +8,7 @@ Given('the FileInput does not have any files', () => {
 })
 
 Then('the onChange handler is called', () => {
-    cy.window().then(win => {
+    cy.window().should(win => {
```

Comment:
This has nothing to do with our PR in particular, but as you're changing the assertion style here and there anyways, it might be a good time to address this:

I think these assertions could be done in a more `cypress`-ish way:
```js
cy.window()
  .its('onChange')
  .should('be.calledOnce')
  .then(stub => stub.getCalls()[0])
  .its('args')
  .should(args => expect(args).to.have.lengthOf(2))
  .and(args => {
    const [payload] = args
    expect(Object.keys(payload)).to.include.members(['files', 'name'])
  })
```

Reply:
Yup, agreed. Same here as in the other comment, should be addressed when fixing this one: https://github.com/dhis2/ui/issues/103#issuecomment-633571972 (see point 1., I think it would fall under there)